src/reducers/sources.js

```diff
@@ -345,6 +345,10 @@ export function getSourcesByURL(state: OuterState, url: string): Source[] {
   return getSourcesByUrlInSources(getSources(state), getUrls(state), url);
 }
 
+export function getSourcesURL(state: OuterState, url: string): Source[] {
```

Comment:
We should combine this function and `getSourcesUrlsInSources` into one; I like the `getSourcesUrlsInSources` name.  `getSourcesURL` doesn't mean anything.